canonical/canonical.com
# WD-12327 Refreshed /solutions/telco

## Done

- Refreshed /solutions/telco
- Added two new sections
## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check if all is in accordance to [copydoc](https://docs.google.com/document/d/1yRSYYY8_UNfZzdLcp_fYxi-STGm40o3UP9vD2vW6brw/edit) and [figma design](https://www.figma.com/design/2QvG4FdcvOpzc9ikdWqKod/Canonical.com%2Fsolutions%2Ftelco?node-id=0-1&t=vGub60kwzbzyIRwr-0).
-  Please check demo [here](https://canonical-com-1292.demos.haus/solutions/telco)

## Issue / Card

Fixes # [WD-12327](https://warthogs.atlassian.net/browse/WD-12327)

## Screenshots



[WD-12327]: https://warthogs.atlassian.net/browse/WD-12327?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
